src/lib/y2storage/proposal/partition_killer.rb

```diff
@@ -49,11 +49,15 @@ def initialize(devicegraph, disks = nil)
       #
       # @param device_sid [Integer] device sid of the partition
       # @return [Array<Integer>] device sids of all the deleted partitions
-      def delete_by_sid(device_sid)
+      def delete_by_sid(device_sid, delete_related_partitions: true)
```

Comment:
please document new parameter

Reply:
and also missing test for that new parameter. ( see only autoinst related tests )

Reply:
Ok, I have improved the documentation.

Regarding the test, we don't have specific tests for `PartitionKiller` class. This is a very internal component, so it is not directly tested. Its behavior is indirectly tested with the proposal tests. And the same happens with the new parameter. The AutoYaST proposal tests are checking that the parameter is actually used.